fiftyone/factory/repo_factory.py

```diff
@@ -15,11 +15,19 @@
     MongoDelegatedOperationRepo,
 )
 
-db: Database = foo.get_db_conn()
+_db: Database = None
+
+
+def _get_db():
+    global _db
+    if _db is None:
+        _db = foo.get_db_conn()
+    return _db
 
 
 class RepositoryFactory(object):
     repos = {}
+    db = None
```

Comment:
nit: is this used?

Reply:
no I think this was a remnant of a different approach. I really didn't like how this class was/is...